pkgs/development/libraries/liblxi/default.nix

```diff
@@ -0,0 +1,35 @@
+{ stdenv, fetchFromGitHub
+, pkgconfig, autoreconfHook
+, libtirpc, rpcsvc-proto, avahi, libxml2
+}:
+
+stdenv.mkDerivation rec {
+  pname = "liblxi";
+  version = "1.13";
+
+  src = fetchFromGitHub {
+    owner = "lxi-tools";
+    repo = "liblxi";
+    rev = "v${version}";
+    sha256 = "129m0k2wrlgs25qkskynljddqspasla1x8iq51vmg38nhnilpqf6";
+  };
+
+  preConfigure = "./autogen.sh";
```

Comment:
Did you test if this is required with autoreconfHook?

Reply:
That preConfigure-line is indeed not required with autoreconfHook.  I've removed it from both liblxi and lxi-tools.